### PR TITLE
add-filtering-capabilities-to-user-and-global-dataspace-file-browsers

### PR DESCRIPTION
**Pull request by oanaBiancaSchiopu**

### CODE REVIEW

File: app/scripts/proactive/view/FileBrowserView.js [hunk 1]
```diff
@@ -113,7 +123,7 @@ define(
             }
             $.ajax({
                 url: that.dataspaceRestUrl + encodeURIComponent(pathname),
-                data: { "comp": "list" },
+                data: { "comp": "list", "includes": '*' + this.filterValue + '*'},
```
Comment (fviale) [inline app/scripts/proactive/view/FileBrowserView.js:126]: I would rather have the filterValue directly applied as includes. The default value of filterValue can be '*'

File: app/scripts/proactive/templates/file-browser-template.html [hunk 2]
```diff
@@ -28,6 +28,7 @@ <h5>
                 <a id="delete-file-btn" href="javascript:void(0)" class="pointer pa-tool-btn" title="Delete the selected file or folder">
                     <i class="fas fa-trash"></i>
                 </a>
+                <label style="padding-left: 160px">Filter: <input type="text" id="filter-files"/> </label>
```
Comment (fviale) [inline app/scripts/proactive/templates/file-browser-template.html:31]: can you add a title attribute to the label "Filter files according to a glob pattern"

**Review by fviale — APPROVED** [review]
LGTM

File: app/scripts/proactive/templates/file-browser-template.html [hunk 3]
```diff
@@ -28,6 +28,7 @@ <h5>
                 <a id="delete-file-btn" href="javascript:void(0)" class="pointer pa-tool-btn" title="Delete the selected file or folder">
                     <i class="fas fa-trash"></i>
                 </a>
+                <label title="Filter files according to a glob pattern" style="padding-left: 160px">Filter: <input type="text" value="*" id="filter-files"/> </label>
```
Comment (mklkun) [inline app/scripts/proactive/templates/file-browser-template.html:31]: glob or global?

Comment (fviale) [reply app/scripts/proactive/templates/file-browser-template.html:31]: it's called a glob pattern.
https://en.wikipedia.org/wiki/Glob_(programming)

Comment (mklkun) [reply app/scripts/proactive/templates/file-browser-template.html:31]: I learned something new. Thanks!

**Pull request merged**